tests/sunpath_test.py

```diff
@@ -62,3 +66,40 @@ def test_leap_year():
     assert sun.datetime.month == 2
     assert sun.datetime.day == 29
     assert sun.datetime.hour == 11
+
```

Comment:
While I made a number of bug fixes here, I am not sure that any of them are going to fix the discrepancy with Revit. When I write the tests today, I will check but I'm not too hopeful that the discrepancy is closed since I think the issue is ultimately that Revit's calculations are not as accurate as ours.